src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Shared/Pages/TodoPage.razor.cs

```diff
@@ -53,8 +53,7 @@ private void FilterViewTodoItems()
     {
         _viewTodoItems = _allTodoItems.Where(todo =>
             _selectedFilter == nameof(AppStrings.Active) ? todo.IsDone == false
-            : _selectedFilter == nameof(AppStrings.Completed) ? todo.IsDone
-            : true
+            : _selectedFilter != nameof(AppStrings.Completed) || todo.IsDone
```

Comment:
please undo this change, it is not related to this issue and the old version is easier to understand.

Reply:
Done